packages/app/obojobo-express/routes/api/events/create_caliper_event_from_client_event_req.js

```diff
@@ -124,6 +124,15 @@ module.exports = req => {
 				sessionIds
 			})
 
+		case 'question:submitResponse':
+			return caliperEvents.createPracticeQuestionSubmittedEvent({
+				actor: actorFromType(ACTOR_USER),
+				draftId: currentDocument.draftId,
+				contentId: currentDocument.contentId,
+				questionId: clientEvent.payload.questionId,
+				sessionIds
+			})
+
```

Comment:
This code (and the related test and snapshot) looks like it's from a different pull request, and snuck its way in somehow.